itou/templates/employee_record/list.html

```diff
@@ -135,6 +143,20 @@ <h5>Nous transférons vos fiches salarié à l'ASP afin de vous faire gagner du
                     </div>
                 {% endif %}
 
+                <div class="row justify-content-end">
+                    <div class="col-auto">
```

Comment:
Évite l’alignement du filtre tantôt à gauche (texte long) tantôt à droite (text court).

```suggestion
                    <div class="col-12 col-md-auto">
```
![Screenshot from 2022-12-05 13-50-19](https://user-images.githubusercontent.com/2758243/205641486-7ebe90f2-a3d1-4974-9618-69ab036e9aec.png)



![image](https://user-images.githubusercontent.com/2758243/205641412-5767a53b-3928-4304-b411-f19f6bca35fe.png)

Reply:
Bien vu :+1: